keyboards/ein_60/keymaps/default/keymap.c

```diff
@@ -0,0 +1,211 @@
+     * |------+------+------+------+------+------|  `------'  |------+------+------+------+------+------|
+     * |      |      |      |      |      |      |            |      |RGBRMO|RGBVAD|RGBSAD|RGBHUD|RGBTOG|
+     * |------+------+------+------+------+------|            |------+------+------+------+------+------|
+     * |      |      |      |      |      |      |            |      |      |      |      |      |BLSTEP|
+     * |------+------+------|--------------------+  ,------.  +--------------------|------+------+------|
+     * |      |      |      |      |      |      |  |      |  |      |      |      |      |      | RESET|
+     * `-----------------------------------------'  `------'  `-----------------------------------------'
+     */
+    [_ADJUST] = LAYOUT( \
+     _______, _______, _______, _______, _______, _______, _______, RGBRST,  RGB_MOD, RGB_VAI, RGB_SAI, RGB_HUI, _______,
+     _______, _______, _______, _______, _______, _______,          _______, RGB_RMOD,RGB_VAD, RGB_SAD, RGB_HUD, RGB_TOG,
+     _______, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, BL_STEP,
+     _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, RESET
+    ),
+    /* Function
+     * ,-----------------------------------------.  ,------.  ,-----------------------------------------.
+     * |      |      |      |      |      |      |  |      |  |      |      |  Up  |      |      |      |
+     * |------+------+------+------+------+------|  `------'  |------+------+------+------+------+------|
+     * |      |      |      |      |      |      |            |      | Left | Down |Right |      |      |
+     * |------+------+------+------+------+------|            |------+------+------+------+------+------|
+     * | Caps |      |      |      |      |      |            |      |      |      |      |      |      |
+     * |------+------+------|--------------------+  ,------.  +--------------------|------+------+------|
+     * |      |      |      |      |      |      |  |      |  |      |      |      |      |      |      |
+     * `-----------------------------------------'  `------'  `-----------------------------------------'
+     */
+    [_FUNCTION] = LAYOUT( \
+     _______, _______, _______, _______, _______, _______, _______, _______, _______, KC_UP,   _______, _______, _______,
+     _______, _______, _______, _______, _______, _______,          _______, KC_LEFT, KC_DOWN, KC_RGHT, _______, _______,
+     KC_CAPS, _______, _______, _______, _______, _______,          _______, _______, _______, _______, _______, _______,
+     _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______
+    )
+};
+
+layer_state_t layer_state_set_user(layer_state_t state) {
+  return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+}
+
+#ifdef OLED_DRIVER_ENABLE
```

Comment:
```suggestion
#ifdef OLED_ENABLE
```

Also, ideally, all of the oled and encoder code should be moved to the keyboard's C file.  Eg: 
https://github.com/qmk/qmk_firmware/blob/a2d43ca024d8e840c882cc72e80e9eb6b70384f1/keyboards/splitkb/kyria/rev1/rev1.c#L58-L148